Cargo.toml

```diff
@@ -8,8 +8,8 @@ authors = ["Bernhard Schuster <bernhard@ahoi.io>",
 
 [dependencies]
 # cuticula = { path = "../cuticula", default-features = false }
-juice = { path = "../juice", default-features = false, version = "0.2.2" }
-coaster = { path = "../coaster", default-features = false, version = "0.1.0" }
+juice = { default-features = false, version = "0.2.2" }
+coaster = { default-features = false, version = "0.1.0" }
 
```

Comment:
That's what I thought, but it still came up after cloning them again. I'm looking into #2 now so I might find something out along the way too.